mindsdb/integrations/handlers/autogluon_handler/README.md

```diff
@@ -0,0 +1,65 @@
+# Autogluon Handler
+
+The Autogluon ML handler integrates Autogluon with MindsDB. You can use it to create and train machine learning models with Autogluon for your existing data. 
+
+## Autogluon
+
+Autogluon is an open-source machine learning library designed to make machine learning easy and accessible. It offers automatic machine learning (AutoML) capabilities, allowing you to quickly build and train machine learning models without requiring in-depth expertise in machine learning.
+
+The Autogluon Python SDK can be found at [https://github.com/autogluon/autogluon](https://github.com/autogluon/autogluon).
+
+## Example Usage
+
+
+
+CREATE ML_ENGINE autoglucon_engine FROM autogluon;
+
+
+SELECT *
+FROM mindsdb.models
+WHERE name='autogluconpredictor_8';
+
+SELECT m.quality
+FROM files.red_wine_quality AS t
+JOIN mindsdb.autogluconpredictor_7 AS m;
+
```

Comment:
Please remove the above CREATE ML_ENGINE and two SELECT statements, as these are duplicated.

Reply:
@martyna-mindsdb  removed it :)